.circleci/config.yml

```diff
@@ -586,7 +586,7 @@ commands:
           google-compute-zone: <<parameters.google-compute-zone>>
           google-project-id: <<parameters.google-project-id>>
       - run:
-          command: gcloud container clusters create ${CLUSTER_ID} --machine-type=n1-standard-8 --cluster-version=<<parameters.gke-version>> --zone=us-central1-c --node-locations=us-central1-c --scopes storage-rw,cloud-platform --num-nodes 1
+          command: gcloud container clusters create ${CLUSTER_ID} --machine-type=n1-standard-8 --cluster-version=<<parameters.gke-version>> --zone=us-central1-a --node-locations=us-central1-a --scopes storage-rw,cloud-platform --num-nodes 1
```

Comment:
If you just want to run the post-commit manually on CI, you just push these changes to a branch of the determined-ai repo (i.e. not a fork) - I assume you're yet proposing this change be permanent for the scheduled runs?

Reply:
assumption is correct. I've rolled the change back. 

I've pushed it as a matter of convenience: if you push the branch to both the fork and the main repo, than circle CI links under this PR in the github UI lead to the circle CI workflow you want to run, and you don't have to go to circle CI to find it.